docs/content/en/docs/how-tos/builders/_index.md

```diff
@@ -9,66 +9,77 @@ to build Docker images.
+* [Dockerfile](https://docs.docker.com/engine/reference/builder/) locally with Docker
 * Dockerfile remotely with [Google Cloud Build](https://cloud.google.com/cloud-build/docs/)
 * Dockerfile in-cluster with [Kaniko](https://github.com/GoogleContainerTools/kaniko)
-* Bazel locally
+* [Bazel](https://bazel.build/) locally
 * [Jib](https://github.com/GoogleContainerTools/jib) Maven and Gradle projects locally
-* [Jib](https://github.com/GoogleContainerTools/jib) Maven and Gradle projects remotely with [Google Cloud Build](https://cloud.google.com/cloud-build/docs/)
+* [Jib](https://github.com/GoogleContainerTools/jib) remotely with [Google Cloud Build](https://cloud.google.com/cloud-build/docs/)
 
 The `build` section in the Skaffold configuration file, `skaffold.yaml`,
-controls how Skaffold builds artifacts. To use a specific tool for building
-artifacts, add the value representing the tool and options for using the tool
-to the `build` section. For a detailed discussion on Skaffold configuration,
-see [Skaffold Concepts: Configuration](/docs/concepts/#configuration) and
+controls how artifacts are built. To use a specific tool for building
+artifacts, add the value representing the tool and options for using that tool
+to the `build` section.
+
+For a detailed discussion on Skaffold configuration, see
+[Skaffold Concepts](/docs/concepts/#configuration) and
 [skaffold.yaml References](/docs/references/yaml).
 
 ## Dockerfile locally with Docker
 
 If you have [Docker Desktop](https://www.docker.com/products/docker-desktop)
-installed on your machine, you can configure Skaffold to build artifacts with
-the local Docker daemon. 
+installed, Skaffold can be configured to build artifacts with the local
+Docker daemon. 
 
 By default, Skaffold connects to the local Docker daemon using
-[Docker Engine APIs](https://docs.docker.com/develop/sdk/). You can, however,
-ask Skaffold to use the [command-line interface](https://docs.docker.com/engine/reference/commandline/cli/)
+[Docker Engine APIs](https://docs.docker.com/develop/sdk/). Skaffold can, however,
+be aked to use the [command-line interface](https://docs.docker.com/engine/reference/commandline/cli/)
 instead. Additionally, Skaffold offers the option to build artifacts with
-[BuildKit](https://github.com/moby/buildkit). After the artifacts are
-successfully built, Skaffold will push the Docker
-images to the remote registry. You can choose to skip this step.
+[BuildKit](https://github.com/moby/buildkit).
 
-To use the local Docker daemon, add build type `local` to the `build` section
-of `skaffold.yaml`.
+After the artifacts are successfully built, Docker images will be pushed
+to the remote registry. You can choose to skip this step.
 
-The `local` type offers the following options:
+### Configuration
+
+To use the local Docker daemon, add build type `local` to the `build` section
+of `skaffold.yaml`. The following options can optionally be configured:
 
 {{< schema root="LocalBuild" >}}
```

Comment:
looking at this rendered it is a bit weird that the default is empty for `push` - shouldn't it be `false` there too? 

[![image](https://user-images.githubusercontent.com/635073/53185778-ae478000-35cd-11e9-85ac-f6904ef6d0ee.png)](http://35.236.109.79:1313/docs/how-tos/builders/)

Reply:
In fact it depends on the k8s context.

Reply:
oh yeah, this is due to a previous merge - I guess it's a more complex logic as currently on skaffold.dev it is manually specified as "false for local clusters, true for remote clusters."...